### PR TITLE
Mes 3910 reuse post debrief page cat be

### PR DESCRIPTION
**Pull request by appleabs**
## Description
Mes 3910 reuse post-debrief page category B+E. Created folder structure for category B+E.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/post-debrief-holding/cat-be/__tests__/post-debrief-holding.cat-be.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { AuthenticationProvider } from '../../../../providers/authentication/authentication';
+import { AuthenticationProviderMock } from '../../../../providers/authentication/__mocks__/authentication.mock';
+import { StoreModule, Store } from '@ngrx/store';
+import { StoreModel } from '../../../../shared/models/store.model';
+import { MockComponent } from 'ng-mocks';
+import { PracticeModeBanner } from '../../../../components/common/practice-mode-banner/practice-mode-banner';
+import { PostDebriefHoldingViewDidEnter } from '../../post-debrief-holding.actions';
+import { PostDebriefHoldingCatBEPage } from '../post-debrief-holding.cat-be.page';
+
+describe('PostDebriefHoldingCatBPage', () => {
```
Comment (MarkPatt89) [inline src/pages/post-debrief-holding/cat-be/__tests__/post-debrief-holding.cat-be.page.spec.ts]: Can you change this description to `PostDebriefHoldingCatBEPage`

Comment (MarkPatt89) [inline src/pages/post-debrief-holding/cat-be/__tests__/post-debrief-holding.cat-be.page.spec.ts]: Can you change this description to `PostDebriefHoldingCatBPage`

File: src/pages/post-debrief-holding/cat-be/post-debrief-holding.cat-be.page.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<ion-content no-bounce id="post-debrief-holding-cat-be-page">
+  <div class="container">
+    <img id='dvsa-logo' src='../../assets/imgs/back-to-office/logo.png' />
+  </div>
+  <div class="container">
+    <button ion-button class="mes-neutral-button office-button top-button" navPush="NonPassFinalisationPage"
```
Comment (MarkPatt89) [inline src/pages/post-debrief-holding/cat-be/post-debrief-holding.cat-be.page.html]: Change navPush to `NonPassFinalisationCatBEPage` as it has been merged to develop. :)

**Review by MarkPatt89 — CHANGES_REQUESTED** [review]
I think you need a module page for both of your new modules (Cat B and Cat B+E which Tom highlighted to me on my last PR, take a look at `NonPassFinalisationPageModule`). A few other small comments.

**Review by MarkPatt89 — APPROVED** [review]
Looks good.

**Review by faggc — APPROVED** [review]

**Review by TomBillingtonUK — APPROVED** [review]
Nice work :)

**Pull request merged**